packages/angular/cli/src/command-builder/utilities/schematic-workflow.ts

```diff
@@ -38,11 +38,15 @@ export function subscribeToWorkflow(
         );
         break;
       case 'update':
-        logs.push(`${colors.cyan('UPDATE')} ${eventPath} (${event.content.length} bytes)`);
+        logs.push(
+          `${colors.cyan('UPDATE')} ${eventPath} (${(event.content as unknown as Buffer).length} bytes)`,
```

Comment:
This does not seem ideal from a type safety perspective.
Is there some information/context on the changes needed here?

Reply:
So as far as I could tell, the types didn't really change but TS is being better at enforcing them. I did start to update the types, but it ended up spanning a ton of files so decided to keep the change simple for now.

Reply:
Done, I've added TODOs on all those `as unknown as` cases.